### DISCUSSION TITLE
icub-firmware-shared version catching up with iCub version?

### DISCUSSION THREAD

**Discussion by kt10aan** [robotology/community#20] [Ideas]
As I had to revert to a previous version of YARP/iCub yesterday, I chose to checkout the tags for the latest package releases (YARP v2.3.63, iCub 1.1.15). Looking for the tags in icub-firmware-shared, the latest one was 1.1.13, which was not working with iCub v1.1.15. I ended up checking out icub-firmware commits from around the date of the iCub v1.1.15 release.

Wouldn't be easier in the future, at the date of each iCub v1.x.y release, to tag the icub-firmware head of the same date as icub-firmware v1.x.y?

**Comment by lornat75** [comment]
This looks like a good suggestion to me.

**Comment by lornat75** [comment]
@mbrunettini: can you update the tagging instructions so we do not forget next time?

**Comment by mbrunettini** [comment]
Fine, tagging instructions updated.
